package.json

```diff
@@ -8,6 +8,59 @@
   },
   "pre-commit": "lint:staged",
   "license": "MIT",
+  "dependencies": {
+    "@uppy-example/aws-companion": "file:examples/aws-companion",
+    "@uppy-example/aws-presigned-url": "file:examples/aws-presigned-url",
+    "@uppy-example/bundled": "file:examples/bundled",
+    "@uppy-example/cdn-example": "file:examples/cdn-example",
+    "@uppy-example/custom-provider": "file:examples/custom-provider",
+    "@uppy-example/dev": "file:examples/dev",
+    "@uppy-example/digitalocean-spaces": "file:examples/digitalocean-spaces",
+    "@uppy-example/multiple-instances": "file:examples/multiple-instances",
+    "@uppy-example/node-xhr": "file:examples/node-xhr",
+    "@uppy-example/php-xhr": "file:examples/php-xhr",
+    "@uppy-example/python-xhr": "file:examples/python-xhr",
+    "@uppy-example/react-example": "file:examples/react-example",
+    "@uppy-example/react-native-expo": "file:examples/react-native-expo",
+    "@uppy-example/redux": "file:examples/redux",
+    "@uppy-example/transloadit": "file:examples/transloadit",
+    "@uppy-example/transloadit-textarea": "file:examples/transloadit-textarea",
+    "@uppy-example/uppy-with-companion": "file:examples/uppy-with-companion",
+    "@uppy-example/xhr-bundle": "file:examples/xhr-bundle",
```

Comment:
Are these `@uppy-example`s here so that we don’t have to add them to Lerna config?

Reply:
They are in the lerna config, but they are also here so their dependencies will also be installed when you do `npm install`

Reply:
Again, missed it, sorry. So because we are not doing `bootstrap` any more, the new rule is every package has to be added to root `package.json`, including examples.

Reply:
dont say sorry! Before examples were not included at all, and you had to manually `npm install` them; we could keep it the same, but I think it'll be nice if you can just clone the repo, do `npm install`, and have everything in place.

Reply:
Ok, I’ll use “my apologies” ;) Yeah, I agree it’s much friendlier if everything is ready after 1 `npm install`.